testing/fuzzing/snmp_transport_fuzzer.c

```diff
@@ -128,9 +128,10 @@ LLVMFuzzerTestOneInput(const uint8_t * data, size_t size)
     /*
      * Cleanup
      */
-    netsnmp_clear_tdomain_list();
-    shutdown_snmp_transport();
+    cleanup:
```

Comment:
Please remove the indentation in front of the `cleanup:` label and preserve the indentation of the last four statements.

Reply:
Done!